index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Catpplication</title>
+    <link rel="stylesheet" href="./src/main.css">
+</head>
+<body>
+    <header>
+        <h1>
+            Please adopt one of us!
+        </h1>
+    </header>
+    
+    <div>
+         <main>
+            <section class="intro">
+                <p>
```

Comment:
the use of the ```<p>``` tags are confusing here as well. Are you using them to make these elements block by default?